client/wcshipping-migration-admin-notice.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import '../assets/stylesheets/migration_to_wcshipping_admin_notice.scss';
+import FeatureAnnouncement from 'components/migration/feature-announcement';
+import ShippingLabel from 'wcs-client/apps/shipping-label';
+import { setNonce, setBaseURL } from 'wcs-client/api/request';
+
+const container = document.getElementById('wcst_wcshipping_migration_admin_notice_feature_announcement');
+const args = container.dataset.args && JSON.parse( container.dataset.args ) || {};
+setNonce( args.nonce );
+setBaseURL( args.baseURL );
+const ShippingLabelStore = ShippingLabel(args);
+const store = createStore(ShippingLabelStore.getReducer(), ShippingLabelStore.getInitialState());
```

Comment:
What are we actually using this for? Do we need to use the reducer?

Reply:
> What are we actually using this for?

The `store` is needed because `<FeatureAnnouncement>` connects to the "label store" for the [isEligableToMigrate ](https://github.com/Automattic/woocommerce-services/blob/1bce661e84bf2efbc27e9bfbdd4cb2896f1f5510/client/components/migration/feature-announcement.jsx#L197-L199)selector.
In order for this selector to work without modification, the store needs to have the `extensions > woocommerce > woocommerceServices > sites >...` path 
![image](https://github.com/Automattic/woocommerce-services/assets/572862/efe5b1e2-ca6f-4c77-b5ce-4b253584eb18)
I could have created a different store by loading a different set of initial states similar to [this](https://github.com/Automattic/woocommerce-services/blob/1bce661e84bf2efbc27e9bfbdd4cb2896f1f5510/client/apps/shipping-label/index.js#L118-L137). But I chose to load the existing store instead.

> Do we need to use the reducer?

No. The `<FeatureAnnouncement>` only consumes the data from the redux store. It doesn't dispatch any action to update the store's state. Reducer isn't needed specifically for this modal. However! `[createStore()](https://redux.js.org/api/createstore)` requires the first arguments to be a reducer. So I use the existing ShippingLabelStore's one. I supposed we could put an empty reducer if we don't want to use the existing one? Something like:
```js
function todos(state = [], action) {
   return state;
}
```

Let me know what you think!